INN/umbrella-sfpublicpress
# Add content-page-classic.php partial along with styles

## Changes

This pull request makes the following changes:

- Adds a `content-page-classic.php` partial for pages to use so they don't look funky

![screencapture-sfpp-test-2020-03-23-16_49_17](https://user-images.githubusercontent.com/18353636/77361757-41e60580-6d26-11ea-96ef-3461d0dfbd1f.png)

## Why



For #61 

## Testing/Questions

Features that this PR affects:

- Pages using the single two column template


Questions that need to be answered before merging:

- [ ] Does everything look ok now?

Steps to test this PR:

1. Create a page and use the two column template
2. View it on desktop and mobile and make sure it looks ok
